conans/test/remove_subsetting_test.py

```diff
@@ -76,7 +74,8 @@ def build(self):
         build_folder = os.path.join(client.current_folder, "build")
         mkdir(build_folder)
         client.current_folder = build_folder
-        client.run('install .. -s os=Windows -s compiler="Visual Studio" -s compiler.version=15 -s arch=x86')
+        client.run('install .. -s os=Windows -s compiler="Visual Studio" -s compiler.version=15 '
+                   '-s arch=x86')
         # This raised an error because build_type wasn't defined
```

Comment:
Check the comment: `this raised an error`, but noone in the test is handling that error (neither before, nor now)...